app/views/preview/components/preview-section-table-row.njk

```diff
@@ -1,11 +1,11 @@
 {% macro previewSectionTableRow(questionId, questionTitle, innerComponent) %}
-  {% if innerComponent %}
+  {% if innerComponent | trim | length > 0 %}
```

Comment:
oooh interesting. Is this what was need to not show when nothing is passed in?

Reply:
Yeah this was one of the ways. Basically the innerComponent was still generating whitespace even though a question did not have an answer.

After our discussion I will add a unit test to check this.